spec/factories/partners/child.rb

```diff
@@ -0,0 +1,14 @@
+FactoryBot.define do
+  factory :partners_child, class: Partners::Child do
+    association :family, factory: :partners_family
+
+    active               { true }
+    archived             { false }
+    comments             { 'Comments ' }
+    date_of_birth        { Time.zone.today - 5.years }
+    first_name           { Faker::Name.first_name }
+    last_name            { Faker::Name.last_name }
+    gender               { Faker::Gender.binary_type }
+    item_needed_diaperid { rand(1..10) }
```

Comment:
We'll probably want to actually match this to an actual `id` someday and add a foreign key constraint. However, I think this is good enough for now :)